pkgs/top-level/aliases.nix

```diff
@@ -1399,6 +1399,7 @@ mapAliases {
   vaultwarden-vault = vaultwarden.webvault; # Added 2022-12-13
   varnish74 = throw "varnish 7.4 is EOL. Either use the LTS or upgrade."; # Added 2024-10-31
   varnish74Packages = throw "varnish 7.4 is EOL. Either use the LTS or upgrade."; # Added 2024-10-31
+  vcprompt = throw "vcprompt has been removed since the source code and homepage are no longer online"; # Added 2025-01-05
```

Comment:
the aur package links to https://github.com/powerman/vcprompt

Reply:
That looks like a promising fork, I must have missed it in my initial search. I'll check if we can use it for our package.

Reply:
Done